config/routes.rb

```diff
@@ -10,6 +10,7 @@
 
   namespace :account do
     get '/dashboard', to: 'dashboard#index'
+    get '/tasks_info_card', to: 'dashboard#tasks_info_card'
```

Comment:
```suggestion
    get '/tasks-info-card', to: 'dashboard#tasks_info_card'
```

please use ```-``` for url instead ```_```